podcasts/ProfileViewController.swift

```diff
@@ -500,7 +500,7 @@ class ProfileViewController: PCViewController, UITableViewDataSource, UITableVie
 
     @objc private func referralsTapped() {
         hideReferralsHint()
-        let viewModel = ReferralSendPassModel(offerInfo: referralsOfferInfo,
+        let viewModel = ReferralSendPassModel(offerInfo: referralsOfferInfo, referralURL: URL(string: "https://pocketcasts.com"),
```

Comment:
Nit: we have a Server constant called `pocketcastsDotCom` -> `ServerConstants.Urls.pocketcastsDotCom`. We use it in few spot like the sharing helper.

Reply:
Good catch, but the URL here will be something defined by the server for each user... so this was a temporary use. But I updated it [here](https://github.com/Automattic/pocket-casts-ios/pull/2166/commits/0f55838c8c48374c404731227546613ec96d25bf) to use the constant for now.